apps/web/utils/actions/group.ts

```
@@ -13,216 +13,227 @@ import { findNewsletters } from "@/utils/ai/group/find-newsletters";
+    }
+  },
+);
+
+export const createPredefinedGroupAction = withActionInstrumentation(
+  "createPredefinedGroup",
+  async (groupId: string) => {
+    if (groupId === NEWSLETTER_GROUP_ID) {
+      return await createNewsletterGroupAction();
+    } else if (groupId === RECEIPT_GROUP_ID) {
+      return await createReceiptGroupAction();
+    }
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)